templates/certified/model-details.html

```diff
@@ -70,9 +70,9 @@ <h2>Release{% if release_details["releases"]|length > 1 %}s{% endif %}</h2>
           {% for release in release_details["releases"] %}
           <li class="{% if release_details['releases']|length > 1 %}p-accordion__group{% endif %}">
             {% if release_details['releases']|length > 1%}
-            <div role="heading" aria-level="3" class="p-accordion__heading">
-              <button type="button" class="p-accordion__tab" style="background-position: top 1.25rem left 1rem;"id="tab{{ loop.index }}" aria-controls="tab{{ loop.index }}-section" aria-expanded="false"><h3 class="u-no-margin--bottom">{{ release.name }}</h3></button>
-            </div>
+            <h3 role="heading" aria-level="3" class="p-accordion__heading u-no-margin--bottom">
```

Comment:
As this is being changed to a `h` element, you don't need the role or aria-level attributes, since they're implicit in a `h3` etc. Also - is the u-no-margin--bottom needed? I don't see that it makes a difference, and the examples don't use it.